SortApp/SortApp.BL/Sortings/Sorter.cs

```diff
@@ -9,6 +11,53 @@ namespace SortApp.BL.Sortings
 	public abstract class Sorter<T> where T : IComparable<T>
 	{
 		/// <summary>
+		/// Adds current iteration to iterations.
+		/// </summary>
+		/// <owner>Oleh Petrenko</owner>
+		/// <param name="arr">
+		/// The current state of the array.
+		/// </param>
+		/// <param name="firstIndex">
+		/// Index of first element which was moved.
+		/// </param>
+		/// <param name="secondIndex">
+		/// Index of second element which was moved.
+		/// </param>
+		protected void AddIteration(T[] arr, int firstIndex, int secondIndex)
+		{
+			Iteration curretnIteration = new Iteration
```

Comment:
I guess Intercation class instances is only created with this three fields set. Maybe we could initialize them from contructor.